resources/electron-preload.js

```diff
@@ -0,0 +1,25 @@
+(function(){
+  'use strict';
+  const {ipcRenderer} = require('electron');
```

Comment:
I don't think destructuring is supported in Node 4. Should this all be written in ES5?